src/bci_build/package.py

```diff
@@ -155,9 +157,16 @@ def pretty_os_version_no_dash(self) -> str:
 
 #: Operating system versions that have the label ``com.suse.release-stage`` set
 #: to ``released``.
-RELEASED_OS_VERSIONS = [OsVersion.SP4, OsVersion.SP5, OsVersion.TUMBLEWEED]
+RELEASED_OS_VERSIONS = [OsVersion.SP5, OsVersion.TUMBLEWEED]
```

Comment:
This causes a regression in the `release_stage` property: https://github.com/SUSE/BCI-dockerfile-generator/blob/9ae6d6a9db1a187b8b26347863d99cd9c7c40e17/src/bci_build/package.py#L482

Reply:
Yeah, I think I can just revert this part. Will try when I am back home